### PR TITLE
Item changes

### PR DESCRIPTION
**Pull request by nesstea**
Updated the SQL with some files for additional effects and/or weapon skills.

### CODE REVIEW

File: scripts/globals/items/excalibur.lua [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        chance = utils.clamp(chance, 5, 95);
+    end
+    if (math.random(0,99) >= chance) then
+        return 0,0,0;
+    else
+		local finalDMG = math.floor(player.getHP(player)/4);
+        if (finalDMG >0) then
+			local physicalResist = target:getMod(MOD_SLASHRES)/1000;
+			finalDMG=finalDMG*physicalResist;
+			target:delHP(finalDMG);
```
Comment (teschnei) [inline scripts/globals/items/excalibur.lua]: still need a call to target:physicaldmgTaken to get the rest of the physical damage modifiers

Comment (nesstea) [inline scripts/globals/items/excalibur.lua]: Will do.

**Comment by maxtherabbit** [comment]
@TeoTwawki that's because finalMagicNonSpellAdjustments() *does* remove hp from the victim

File: scripts/globals/items/shiranui.lua [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			params.bonusmab = 0;
+			params.includemab = true;
+			lightDamage = addBonusesAbility(player, ELE_LIGHT, target, lightDamage, params);
+			lightDamage = lightDamage * applyResistanceAddEffect(player,target,ELE_LIGHT,0);
+			lightDamage = adjustForTarget(target,lightDamage,ELE_LIGHT);
+			if (lightDamage < 0) then
+				lightDamage = 0
+			end
+			lightDamage = finalMagicNonSpellAdjustments(player,target,ELE_LIGHT,lightDamage);
+			if (lightDamage >35) then
```
Comment (maxtherabbit) [inline scripts/globals/items/shiranui.lua]: which reminds me...you need to cap this valuebefore calling finalMagicNonSpellAdjustments()

**Comment by maxtherabbit** [comment]
and we need to handle the victim's phalanx/stoneskin for the excalibur hits

add in something like this before the delHP() call:

finalDMG = finalDMG - target:getMod(MOD_PHALANX);
utils.clamp(finalDMG, 0, 99999);
finalDMG = utils.stoneskin(target, finalDMG);

File: scripts/globals/items/shiranui.lua (hunk 2 above)
Comment (nesstea) [inline scripts/globals/items/shiranui.lua]: Nrrh is getting the actual cap from retail.

**Comment by teschnei** [comment]
oh yeah, you have to remove all the delStatusEffect lines in the additional effects of your weapon procs

**Comment by TeoTwawki** [comment]
eva down removed eva boost, def down removed def boost. And take out the if(not effect check, it will overwrite or not by power lv

**Comment by TeoTwawki** [comment]
I made same mistake earlier, bad copy paste

**Comment by TeoTwawki** [comment]
#1536 I believe I fixed all the erasing of wrong status effects (def down will remove def boost, rather than itself, unless the previous instance of it was weaker)

**Pull request merged**